ui/tests/acceptance/settings/mount-secret-backend-test.js

```diff
@@ -19,11 +28,15 @@ import logout from 'vault/tests/pages/logout';
 import mountSecrets from 'vault/tests/pages/settings/mount-secret-backend';
 import { mountableEngines } from 'vault/helpers/mountable-secret-engines'; // allEngines() includes enterprise engines, those are tested elsewhere
 import { supportedSecretBackends } from 'vault/helpers/supported-secret-backends';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { SELECTORS as OIDC } from 'vault/tests/helpers/oidc-config';
+import { adminOidcCreateRead, adminOidcCreate } from 'vault/tests/helpers/secrets/policy-generator';
 
 const consoleComponent = create(consoleClass);
 
 // enterprise backends are tested separately
 const BACKENDS_WITH_ENGINES = ['kv', 'pki', 'ldap', 'kubernetes'];
+const WIF_SECRET_ENGINES = ['aws']; // eventually this will include Azure and GCP.
```

Comment:
Same comment here, I think moving this to a util would make sense. Or maybe adding to the existing helper: https://github.com/hashicorp/vault/blob/main/ui/app/helpers/mountable-secret-engines.js#L0-L1

as another const